morello/morello.md

```diff
@@ -117,87 +105,74 @@ reserved.
 The following support level definitions are used by the ACLE
 specifications:
 
-**Release**
+_**Release**_
```

Comment:
I see you have added the `_` to render italic as the original rendering. This was the requirement of the fix in the PR, but for this specific one I think we are good anyway to leave it as just bold text without the italic. Please restore it.